src/com/content-games/games.js

```diff
@@ -166,55 +164,34 @@ export default class ContentGames extends Component {
 //        props.class.push("content-item-boxes");
 
 		var LoadMore = null;
-		const Games = [];
+		var Games = [];
```

Comment:
I would use `let` rather than `var` so scope of variables is easier to read and predict

Reply:
Good call.